frontend/server/src/DAO/Problems.php

```diff
@@ -1154,15 +1154,7 @@ public static function hasBeenUsedInCoursesOrContests(
                 GROUP BY
                     t.name;";
 
-        $result = [];
-        /** @var array{alias: string, problems_per_tag: int} $row */
-        foreach (
-            \OmegaUp\MySQLConnection::getInstance()->GetAll(
-                $sql
-            ) as $row
-        ) {
-            $result[] = new \OmegaUp\DAO\VO\Problems($row);
-        }
-        return $result;
+        /** @var array{name: string, problems_per_tag: int} */
```

Comment:
```suggestion
        /** @var list<array{name: string, problems_per_tag: int}> */
```

esto regresa una lista, así que necesita declarar el hecho de que es una lista. y L1137 tiene que tener la misma anotación.